Ginger/Ginger/SolutionWindows/TreeViewItems/ApplicationModelsTreeItems/AppApiModelsFolderTreeItem.cs

```diff
@@ -37,7 +37,7 @@ public class AppApiModelsFolderTreeItem : NewTreeViewItemBase, ITreeViewItem
     {
         public RepositoryFolder<ApplicationAPIModel> mAPIModelFolder;
         private APIModelsPage mAPIModelsPage;
-        private ObservableList<ApplicationAPIModel> mChildAPIs = null;
+        public ObservableList<ApplicationAPIModel> mChildAPIs = null;
```

Comment:
why this is needed as Public?

Reply:
was trying something. Later, with new approach, it's no more needed, thus, undone the change